mbed_lstools/lstools_ubuntu.py

```diff
@@ -15,297 +15,14 @@
 limitations under the License.
 """
 
-import re
-import subprocess
+from lstools_linux_generic import MbedLsToolsLinuxGeneric
 
-from lstools_base import MbedLsToolsBase
 
-
-class MbedLsToolsUbuntu(MbedLsToolsBase):
+class MbedLsToolsUbuntu(MbedLsToolsLinuxGeneric):
```

Comment:
is this needed at all, because MbedLsToolUbuntu is now direct inheritance without any special functionality ?